app/_locales/en/messages.json

```diff
@@ -769,7 +769,8 @@
     "message": "Bitcoin"
   },
   "bitcoinSupportToggleDescription": {
-    "message": "Turning on this feature will give you the option to add a Bitcoin Account to your MetaMask Extension derived from your existing Secret Recovery Phrase. This is an experimental Beta feature, so you should use it at your own risk."
+    "message": "Turning on this feature will give you the option to add a Bitcoin Account to your MetaMask Extension derived from your existing Secret Recovery Phrase. This is an experimental Beta feature, so you should use it at your own risk. To give us feedback on this new Bitcoin experience, please fill out this $1.",
```

Comment:
"fill out this form: $1" maybe?

Maybe this was the original text, so we can keep it as is if yes.

Reply:
But in this case $1 would be the word "form" with the link, like `[form](survey_link)`, so I'm still not sure if we're in the same page

Reply:
Nope all good, I was not looking at the actual "expansion" in the other file.

Thanks for explaining :)

Reply:
Ah I see what you mean, I could be good from a security perspective but then we need Alex to approve it. Lets keep it like this